vaadin-renderer-flow-parent/vaadin-renderer-flow/src/main/java/com/vaadin/flow/data/renderer/BasicRenderer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2000-2022 Vaadin Ltd.
+ * Copyright 2000-2023 Vaadin Ltd.
```

Comment:
nit: NativeButtonRenderer.java still says 2022

Reply:
Took the opportunity to update this for all files in 'vaadin-renderer-flow'